### DISCUSSION TITLE
Spawning west as civilian

### DISCUSSION THREAD

**Discussion by CLAFMB** [Vdauphin/HeartsAndMinds#1614] [Question and answer]
A ultima versão, quando edito ela e depois vou jogar no mp, ela spawna o blufor como civil, mesmo na linha de script estando como west.
alguem sabe como resolver isso?
![Arma3_x64 2024-05-09 05-34-13](https://github.com/Vdauphin/HeartsAndMinds/assets/169346827/3555073d-a318-4550-9b1d-f06bfa8cf8b2)

**Comment by Vdauphin** [comment]
Hello,

Can you explain in English?

Cheers

**Comment by CLAFMB** [comment]
Hi, this issue of the entity turning into a civilian instead of a blufor I solved it, but I have a much bigger problem, I edit the game mode and for some reason it freezes after about 2 hours of playing the mission it freezes I can walk around the map but my friends appear static, I've already made the corrections suggested in other posts I found here on the site, I don't know what else to do.

**Reply by Vdauphin** [reply]
Just download the mission again and test each change you make
Also check your .rpt and fix error you see
